docs/src/misc.rst

```diff
@@ -291,6 +305,23 @@ API
 
     .. versionadded:: 1.9.0
 
+.. c:function:: int uv_get_passwd(uv_passwd_t* pwd)
```

Comment:
can we bikeshed a bit on the naming? :-) how about `uv_os_get_passwd` as in uv_namespace_xyz?